kie-api/src/main/java/org/kie/api/event/process/SignalEvent.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.api.event.process;
+
+/**
+ * An event when a signal is thrown
+ */
+public interface SignalEvent extends ProcessNodeEvent {
+    /**
```

Comment:
add a string for implementation

Reply:
not doing it, currently skipping information from process, to be done with another JIRA